base/compiler/ssair/ir.jl

```diff
@@ -319,6 +319,15 @@ function setindex!(x::IRCode, @nospecialize(repl), s::SSAValue)
     return x
 end
 
+function setindex!(x::IRCode, repl::Instruction, s::SSAValue)
```

Comment:
This method definition may prohibit inlining and result in dynamic dispatches, since now `setindex!(::IRCode, ::Any, ::SSAValue)` won't be resolved to a single matching method.
Can we remove either of those definitions?

Reply:
Do we even use this method now?